android/src/main/java/com/swmansion/rnscreens/RNSUtil.kt

```diff
@@ -0,0 +1,5 @@
+package com.swmansion.rnscreens
+
+public inline fun <T> Iterable<T>.forEachSatisfying(predicate: (T) -> Boolean, action: (T) -> Unit) {
```

Comment:
Is this used anywhere?

Reply:
[7aad6e6](https://github.com/software-mansion/react-native-screens/pull/1558/commits/7aad6e6a32178303c9905760644a2cd748911077)